tests/services/test_results_expand.py

```diff
@@ -45,6 +45,10 @@ def read_many(self, identity, ids):
 
 
 class CreatedByExpandableField(ExpandableField):
+    def tombstone(self, value):
```

Comment:
remove all implementations of `tombstone` when the parent class is not ABC anymore